### ISSUE TITLE
Fedora 35 - wont authenicate

### ISSUE THREAD

**Issue by TheHelloDev** [boltgolt/howdy#600]
I've setup howdy and running sudo/logging in (from anywhere) won't work.
I'm using gnome 41 fedora 35 beta.
I'm pretty sure Howdy doesn't have permissions, but fedora doesn't have the 'auth' command, so the fix from the arch wiki will not work.
I gave it 755 through chmod (/lib64/security/howdy/) 
My webcam's led doesn't even turn on, unless I'm adding a face or running `sudo howdy test`

----
I've searched for similar issues already, and my issue has not been reported yet. (or I just can't find it.)

Linux distribution (if applicable): fedora 35

Howdy version (`sudo howdy version`): 2.6.0

**Comment by kuba21333** [comment]
https://copr.fedorainfracloud.org/coprs/principis/howdy/ follow the instructions here it works, but when launching terminal you have to type in your password for some reason

**Comment by cokoghenun** [comment]
It worked for sudo on f35 but didn't work for gdm

**Comment by Poloin34** [comment]
> https://copr.fedorainfracloud.org/coprs/principis/howdy/ follow the instructions here it works, but when launching terminal you have to type in your password for some reason

Working but.. only in terminal. 
Without restarting, after logout, it show "Unknown error 1" and after restart, lol, I'm 100% unable to login, it show the error + show another when entering password.... Had to backup my gdm- file with a live usb to login

So right now, I can only use Howdy in terminal (maybe apps, but not tested) but not for login, which would have been my main use..

**Comment by TheHelloDev** [comment]
Not using Fedora anymore, or howdy.

**Comment by FIUSHerson** [comment]
I found something that worked for me!

In the [arch wiki](https://wiki.archlinux.org/title/Howdy), it tells us to to insert `auth sufficient pam_python.so /lib/security/howdy/pam.py` into /etc/pam.d/sudo.

However, /lib doesn't exist in Fedora as it has been replaced with /lib64. So, all you need to do is change up the path a little bit.

**TL;DR**: Insert `auth sufficient pam_python.so /lib64/security/howdy/pam.py` into the second line of /etc/pam.d/sudo

**Comment by cokoghenun** [comment]
> I found something that worked for me!
> 
> In the [arch wiki](https://wiki.archlinux.org/title/Howdy), it tells us to to insert `auth sufficient pam_python.so /lib/security/howdy/pam.py` into /etc/pam.d/sudo.
> 
> However, /lib doesn't exist in Fedora as it has been replaced with /lib64. So, all you need to do is change up the path a little bit.
> 
> **TL;DR**: Insert `auth sufficient pam_python.so /lib64/security/howdy/pam.py` into the second line of /etc/pam.d/sudo

Thanks man, I'll try it out
